.drone.yml

```diff
@@ -133,7 +133,7 @@ steps:
       - echo "server stopped"
 
   - name: setup
-    image: node:14-alpine
+    image: node:14
```

Comment:
Do we still need `node:14` now that `curl` is being installed manually?

Reply:
We do not. I've fixed this in https://github.com/webkom/lego-webapp/tree/ollfkaih/master. @ollfkaih do you want to update your branch with my newest changes?

Reply:
Tror det ble riktig nå, evt. dobbeltsjekk hashen i drone.yml